### PR TITLE
re: #163 - Fixed Capitol One

### PR DESCRIPTION
**Pull request by stepps00**
Updated Capitol One and Pioneer Square records. Capitol One geometry was absorbed into Pioneer Squre record, deprecated, and superseded fields updated.

### CODE REVIEW

File: data/858/685/17/85868517.geojson [hunk 1]
```diff
@@ -52,6 +54,7 @@
     ],
     "src:lbl:centroid":"yerbashapes",
     "wof:belongsto":[
+        85866047,
```
Comment (nvkelso) [inline data/858/685/17/85868517.geojson]: Not sure about this. @thisisaaronland If we're superseding this record into `85866047`, do we need to include `85866047` in `"wof:belongsto"`?

**Comment by nvkelso** [comment]
Good overall! Couple things to update:

1. You need to exportify the `wof:id=85866047` record to recalculate the bounding box and other fields.

2. Let's also mark `wof:id=85866047` for **Pioneer Square** with `"mz:is_current": 1,` since we're confirming the existence of this neighbourhood. (You already marked `wof:id=85868517` with `"mz:is_current": 0,`.)

**Tip:** Confirming your proposal, Nat noted this Capitol One "neighbourhood" feature is really a landuse / AOI feature by marking it: `u'mz:is_landuse_aoi': u'1',`.

**Comment by nvkelso** [comment]
Verified fixes for 1 and 2 above in https://github.com/whosonfirst/whosonfirst-data/pull/268/commits/fb95e0a3c69f392881b08191d316d74689403b32 and https://github.com/whosonfirst/whosonfirst-data/pull/268/commits/563522d7f775242befe8499d09f98a9ac10ead0a.

**Comment by stepps00** [comment]
* Exportify takes care of the `wof:belongsto` field. 
* Also, not necessary to include the superseded record to this record's `wof:belongsto` field. 
* Reverted edits to `wof:belongsto` field.

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**